modules/dnn/src/dnn.cpp

```diff
@@ -3543,6 +3543,9 @@ Net Net::Impl::createNetworkFromModelOptimizer(InferenceEngine::CNNNetwork& ieNe
                 }
                 else
                 {
+#if INF_ENGINE_VER_MAJOR_GT(INF_ENGINE_RELEASE_2020_4)
+                    CV_Error(Error::StsNotImplemented, "This OpenCV version is built without Inference Engine IR v7 support");
```

Comment:
The message seems like there is an option to build with IRv7 support. I'd like to recommend to rephrase that support of old IRs has been removed.

Reply:
It's just a copy-paste from the message for NN Builder :)

Reply:
> The message seems like there is an option to build with IRv7 support

Just to pass older version of Inference Engine :)

Reply:
fixed